### PR TITLE
feature/WAL-642

### PR DESCRIPTION
**Pull request by BtencateSphereon**
some commits ended up in WAL-641 which this branch is based off. asked sadjad to revert that so we get decent PR's and reviews

### CODE REVIEW

File: packages/oid4vci-demo-frontend/src/configs/coc.json [hunk 1]
```diff
@@ -0,0 +1,245 @@
+{
+  "general": {
+    "credentialName": "Sphereon Guest",
```
Comment (nklomp) [inline packages/oid4vci-demo-frontend/src/configs/coc.json]: Please merge and look at develop for proper values in this section

**Comment by nklomp** [comment]
Please merge in develop and adjust your VCI metadata if the merge doesn't already take care of it

**Review by nklomp — CHANGES_REQUESTED** [review]
Please merge in develop and adjust your VCI metadata if the merge doesn't already take care of it

File: packages/oid4vci-demo-frontend/src/configs/coc.json (hunk 1 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/configs/coc.json]: metadata has been changed to:

"general": {
    "credentialName": "CoC credential",
    "issueCredentialType": "KvkInschrijving",
    "baseUrl": "https://kvk.demo.sphereon.com/agent",
    "verifierUrl": "https://sphereon.com",
    "verifierUrlCaption": "CoC credential",
    "downloadUrl": "https://kvk.demo.sphereon.com/agent/#/download"
  },

Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/configs/coc.json]: not sure about the verifierUrl

Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/configs/coc.json]: updated verifierUrl to https://kvk.demo.sphereon.com/verifier

Comment (nklomp) [reply packages/oid4vci-demo-frontend/src/configs/coc.json]: Please change CoC credential for both to KvK Inschrijving. Domain values should all be in Dutch

Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/configs/coc.json]: updated

**Review by nklomp — APPROVED** [review]

**Comment by nklomp** [comment]
PR approved

**Pull request merged**